dev/stage-binaries.sh

```diff
@@ -17,10 +17,16 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-
+SCALA_VERSION=2.12
 FLINK_VERSIONS=1.12,1.13,1.14
 SPARK_VERSIONS=2.4,3.0,3.1,3.2
 HIVE_VERSIONS=2,3
 
-./gradlew -Prelease -DflinkVersions=$FLINK_VERSIONS -DsparkVersions=$SPARK_VERSIONS -DhiveVersions=$HIVE_VERSIONS publishApachePublicationToMavenRepository
+./gradlew -Prelease -DscalaVersion=$SCALA_VERSION -DflinkVersions=$FLINK_VERSIONS -DsparkVersions=$SPARK_VERSIONS -DhiveVersions=$HIVE_VERSIONS publishApachePublicationToMavenRepository
+
+# Also publish Scala 2.13 Artifacts for versions that support it.
+# Flink does not yet support 2.13 and Hive doesn't need a Scala specification.
+SCALA_VERSION=2.13
+SPARK_VERSIONS_2.13=3.2
+./gradlew -P release  -DscalaVersion=$SCALA_VERSION -DsparkVersions=$SPARK_VERSIONS_2.13 -DhiveVersions= -DflinkVersions= publishApachePublicationToMavenRepository
```

Comment:
I think that this needs to exclude all of the non-Spark libraries, like we do in the workflows that just run Spark tests. That would look like this:

```bash
./gradlew -P release :iceberg-spark:iceberg-spark-3.2_2.13:publishApachePublicationToMavenRepository :iceberg-spark:iceberg-spark-extensions-3.2_2.13:publishApachePublicationToMavenRepository :iceberg-spark:iceberg-spark-runtime-3.2_2.13:publishApachePublicationToMavenRepository
```

You could parameterize that, but it may be worth just adding it and we can worry about updating it when we have more versions later.

Reply:
Updated to just be this call directly for now, with the comments but without the updated template variables.